suncity/create_star-fusion_resources.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+# This is a custom script for the coyote pipeline as it does not have a plug-n-play
+# ctat library available from broad. As such we are building from scratch.
+# Usage: create_star-fusion_resources.sh <Config.ini>
+
+### Setting as an interactive BASH session and forcing history to capture commands to a log/README file
+HISTFILE=~/.bash_history
+set -o history
+set -ue
+
+# Check resources.ini was provided on the command line
+if [ -n "$1" ]
+then
+  echo "Required ini file detected"
+else
+  echo "Input INI file not provided, exiting due to missing requirement"
+  exit 1
+fi
+
+# Read required variables from configuration file
+. ${1}
+
+####################################
+## Navigate Directory Structure
+###################################
+
+# Check top level directory if not available
+if [ -e ${TOPLEVEL_DIR} ]
+then
+    echo "Top level directory: ${TOPLEVEL_DIR} exists, moving into it"
+    cd ${TOPLEVEL_DIR}
+else
+    echo "Top level directory NOT found, IT IS REQUIRED, EXITING"
+    exit 1
+fi
+
+# Check gene_model directory if not available
+if [ -e gene_model ]
+then
+    echo "Gene Model directory exists, moving into it"
+    cd gene_model
+else
+    echo "Gene Model directory NOT found, IT IS REQUIRED, exiting as it should be created by a previous step"
+    exit 1
+fi
+
+# Check specific gene model directory
+if [ -e ${GENE_MODEL_NAME} ]
+then
+    echo "Specific Gene Model directory exists, moving into it"
+    cd ${GENE_MODEL_NAME}
+else
+    echo "Specific Gene Model directory NOT found, IT IS REQUIRED, exiting as it should be created by a previous step"
+    exit 1
+fi
+
+# Check that the required GTF was created successfully
+if [ -e GENE_MODEL_GTF_GENERATION_COMPLETE ]
+then
+    echo "Required gene model GTF exists, moving forward"
+else
+    echo "Required gene model GTF DOES NOT exist, exiting"
+    exit 2
+fi
+
+# Make gene_model specific tool_resources directory if not available
+if [ -e tool_resources ]
+then
+    echo "Gene Model tool_resources directory exists, moving into it"
+    cd tool_resources
+else
+    echo "Gene Model tool_resources directory NOT found, creating and entering it now"
+    mkdir tool_resources
+    cd tool_resources
+fi
+
+# Make star-fusion directory if not available
+if [ -e "starFusion_${GENE_MODEL_NAME}" ]
+then
+    echo "starFusion directory exists, moving into it"
+    cd starFusion_${GENE_MODEL_NAME}
+else
+    echo "starFusion directory NOT found, creating and moving into it now"
+    mkdir starFusion_${GENE_MODEL_NAME}
+    cd starFusion_${GENE_MODEL_NAME}
+fi
+
+# Make a build folder
+mkdir -p starFusion_Resources
+cd starFusion_Resources
+
+# Create symbolic link
+REFERENCE_RNA_GENOME_FASTA=${TOPLEVEL_DIR}/genome_reference/${REFERENCE_RNA_GENOME_NAME}
```

Comment:
Doesn't Suncity use the same reference genome for RNA and DNA? this looks like legacy code from phoenix